arches_lingo/settings.py

```diff
@@ -54,18 +159,18 @@
 UPLOADED_FILES_DIR = "uploadedfiles"
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "--+c7*txnosqv=flep00qp+=t-xhrj%f4==r8w*n_7pm@mi%)7"
-
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+SECRET_KEY = get_optional_env_variable(
+    "ARCHES_SECRET_KEY", "--+c7*txnosqv=flep00qp+=t-xhrj%f4==r8w*n_7pm@mi%)7"
```

Comment:
Might be nice to reuse the `django-insecure-` prefix so that anybody who tries to deploy lingo themselves and forgets to rotate this gets the built-in check, see archesproject/arches@a6d886025e4cb4cc7aa032e1a63b125c48990643, but let me know if I'm missing context

Reply:
good point.